src/walletSdk/index.ts

```diff
@@ -75,20 +75,10 @@ export class Wallet {
     return new Recovery({
       cfg: this.cfg,
       stellar: this.stellar(),
-      httpClient: this.getClient(httpClientConfig),
+      httpClient: this.cfg.app.defaultClient,
       servers
     });
   }
-
-  getClient(httpClientConfig: AxiosRequestConfig = {}): AxiosInstance {
```

Comment:
originally this method was added because kotlin had it. After looking at it again it's because the kotlin sdk needs to create and store multiple client instances so it can close them later, which typescript sdk doesn't have to do. So removing this method

Reply:
@Ifropc lmk if my understanding of why the kotlin sdk has this is wrong